src/electionguard_cli/e2e_steps/input_retrieval_step.py

```diff
@@ -15,6 +13,37 @@
 class InputRetrievalStep(E2eStepBase):
     """Responsible for retrieving and displaying user provided inputs."""
 
+    ###########################
+    # Public Mehtods
```

Comment:
*Methods

Reply:
These are frowned upon for the record. 

Just an example:
https://stackoverflow.com/questions/34259939/is-it-pythonic-to-have-header-comments-in-a-python-script